src/lib/rollup.index.ts

```diff
@@ -48,6 +51,9 @@ const SynapseComponents = {
   SynapseHomepage,
   EntityFinder,
   ErrorPage,
+  DownloadCartPage,
+  ShowDownloadV2,
+  DownloadConfirmation
```

Comment:
Exported for SWC.

Reply:
(replacing the SWC implementation of the DownloadConfirmation with our SRC one!)